### PR TITLE
Pull Request Template

### PR DESCRIPTION
**Pull request by Roseywinter**
This a pull request template draft... looking for some feedback. :)

-  How does the language and wording seem to you?
-  What do you think about the order of sections? Should they be re-arranged?
-  What do you think about the sections? Do you want to remove any? Add any?
-  Feel free to add any other thoughts/notes about the PR Template! Thanks!

(NOTE: will remove file to .github if we want to keep file hidden from root directory)

### CODE REVIEW

**Review by aquaomarine — APPROVED** [review]

**Review by discombobulateme — CHANGES_REQUESTED** [review]
Hey! How hard is it to create a template? 😅
Some notes:

- the template itself should be straightforward, help people to be as direct and informative as possible to help the reviewer. Think about placeholders that help you start writting
- all the explanation content should be in the contribution guidelines, which people should read before contributing and making a PR
- Checklists are awesome! Type of contribution. One thought on that one... shall we have different templates for different types of contributions? [This example ](https://github.com/forem/forem/blob/main/.github/PULL_REQUEST_TEMPLATE.md)uses one single generic template and you use the checkbox so say which type. It bothers me that it gets toooooo texty but, [this example](https://dev.to/opensauced/how-to-create-a-good-pull-request-template-and-why-you-should-add-gifs-4i0l), although texty, its nicer, as you are doing, it adds graphics
- [This](https://axolo.co/blog/p/part-3-github-pull-request-template) gives different templates structure to different types of template
- On the 2nd checklist, the 2 last items are unnecessary. You do not ship broken code, never. It can be unfinished, but it means its out of the scope. Draft is a function, it will not be reviewed anyway while a draft.  Check the link above to see other checklists

my heart for including accessibility <3

**Comment by Roseywinter** [comment]
Made some changes to the PR Template. Is it coming along better @discombobulateme ?

Changes made:
- Removed some excess wording, and commented out some lines.
- Improved some of the checklist flows
- Included section about testing
- Improved section about related issues

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+- [ ] Other _(please replace this text with details)_
+
+## 🚀 Description
+
+<!--- Describe your changes in detail -->
+
+## 🤝 Related Issue
+
+-  This PR is related to Issue #
+-  This is a linked PR that will close Issue #
```
Comment (discombobulateme) [inline .github/pull_request_template.md:16]: Delete this

**Review by discombobulateme — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (Roseywinter) [reply .github/pull_request_template.md]: Done! :)

**Pull request merged**